code/__DEFINES/misc.dm

```diff
@@ -253,7 +253,8 @@
 #define MATRIX_DEFAULT list(1, 0, 0, 0,\
                             0, 1, 0, 0,\
                             0, 0, 1, 0,\
-                            0, 0, 0, 1)
+                            0, 0, 0, 1,\
+                            0, 0, 0, 0)
```

Comment:
This matrix, as it's used, doesn't actually need to exist. Anything set to this matrix can just be set to `null` to get default colors.

Also, every matrix used for client colors only needs to be 3x3, because only RGB is supported. This particular matrix is used for setting atom colors in the animation library, which *can* have alpha, but it doesn't need to be used there either.